platform_plugin_communications/api/views.py

```diff
@@ -0,0 +1,109 @@
+"""API Views for the plugin platform-plugin-communications."""
+import datetime
+import json
+import logging
+
+import dateutil
+import pytz
+from django.db import transaction
+from django.http import HttpResponseBadRequest, JsonResponse
+from django.views.decorators.cache import cache_control
+from django.views.decorators.csrf import ensure_csrf_cookie
+from django.views.decorators.http import require_POST
+from opaque_keys.edx.keys import CourseKey
+
+import platform_plugin_communications.utils as task_api
+from platform_plugin_communications.edxapp_wrapper.bulk_email import create_course_email, is_bulk_email_feature_enabled
+from platform_plugin_communications.edxapp_wrapper.course_overviews import get_course_overview_or_none
+from platform_plugin_communications.edxapp_wrapper.instructor_views import (
+    _get_branded_email_from_address,
+    _get_branded_email_template,
+    common_exceptions_400,
+    permissions,
+    require_course_permission,
+    require_post_params,
+)
+
+log = logging.getLogger(__name__)
+
+
+@transaction.non_atomic_requests
+@require_POST
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_course_permission(permissions.EMAIL)
+@require_post_params(
+    send_to="sending to whom", subject="subject line", message="message text"
+)
+@common_exceptions_400
+def send_email_api_view(request, course_id):
+    """
+    Get the forum email notification preference for the user.
+
+    Extracted from: lms.djangoapps.instructor.views.api import send_email
+    """
+    return send_email(request, course_id)
+
+
+def send_email(request, course_id):
+    """
+    Send email to individual learners.
+    """
```

Comment:
This docstring is not accurate. Also, can we add more detail? The function does a lot of stuff for such a short description